src/components/Modals/WithdrawalStatusModal/WithdrawalStatusModal.tsx

```diff
@@ -31,7 +31,9 @@ export default class WithdrawalStatusModal extends React.PureComponent<Props> {
     }
     const { status, amount } = withdrawal
     const isWithdrawalPending = status === WithdrawalStatus.PENDING
-    const isWithdrawalCheckpoint = status === WithdrawalStatus.CHECKPOINT
+    const isWithdrawalCheckpoint =
+      status === WithdrawalStatus.CHECKPOINT ||
+      status === WithdrawalStatus.COMPLETE
```

Comment:
Is this intended or leftover from testing?

Reply:
nevermind, as discussed offline now i get that this is to mark the checkbox as checked